vs2015/install_activate.bat

```diff
@@ -4,6 +4,18 @@ set VER=14
 mkdir "%PREFIX%\etc\conda\activate.d"
 COPY "%RECIPE_DIR%\activate.bat" "%PREFIX%\etc\conda\activate.d\vs%YEAR%_compiler_vars.bat"
 
+echo if "%%VC140_ON_VS2017%%" == "" ( >> "%PREFIX%\etc\conda\activate.d\vs%YEAR%_compiler_vars.bat"
+
+IF "%cross_compiler_target_platform%" == "win-64" (
+  echo SET "CMAKE_GENERATOR=Visual Studio 15 2017 Win64" -T "v140" >> "%PREFIX%\etc\conda\activate.d\vs%YEAR%_compiler_vars.bat"
```

Comment:
I'm not sure about the quoting here.  I'm not sure that the -T "V140" will get included properly.  I think the proper quoting *might* be:

SET "CMAKE_GENERATOR=""Visual Studio 15 2017 Win64"" -T v140"

Reply:
We use `cmake -G "%CMAKE_GENERATOR"` and with my quoting this turns into
`cmake -G "Visual Studio 15 2017 Win64" -T "v140"`

with yours, it's
`cmake -G """Visual Studio 15 2017 Win64"" -T v140"`

Reply:
Cool, thanks for checking.